src/go/k8s/controllers/redpanda/console_controller.go

```diff
@@ -101,7 +101,22 @@ func (r *ConsoleReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 }
 
 func (r *ConsoleReconciler) createIngress(ctx context.Context, cluster *redpandav1alpha1.Cluster, console *redpandav1alpha1.Console, log logr.Logger) error {
-	return nil
+	subdomain := ""
+	proxyAPIExternal := cluster.PandaproxyAPIExternal()
+	if proxyAPIExternal != nil {
+		subdomain = fmt.Sprintf("%s.%s", "console", proxyAPIExternal.External.Subdomain)
+	}
+	// If subdomain is empty, no Ingress will be created.
+	ingress := resources.NewIngress(
+		r.Client,
+		console,
+		r.RuntimeScheme,
+		subdomain,
+		console.GetName(),
+		"http",
```

Comment:
This should be connected with https://github.com/RafalKorepta/redpanda/blob/e812d191feb43a49fa090fb6aa15d9ae9d7e1cbd/src/go/k8s/controllers/redpanda/console_controller.go#L124

Could you create const and use in both places?

Reply:
Updated